pkgs/top-level/darwin-packages.nix

```diff
@@ -64,6 +64,8 @@ impure-cmds // appleSourcePackages // chooseLibs // {
 
   inherit apple_sdk apple_sdk_10_12 apple_sdk_11_0;
 
+  asitop = pkgs.python3Packages.callPackage ../os-specific/darwin/asitop { };
```

Comment:
Darwin-specific packages could still be placed in `all-packages.nix`.

Reply:
Moved it!

Reply:
And `darwin.asitop` -> `asitop` in the commit message.

Reply:
Done, sorry for that.